tempesta_fw/http_parser.c

```diff
@@ -70,7 +83,7 @@ int __fsm_const_state;							\
 int __fsm_n __attribute__((unused));					\
 size_t __fsm_sz __attribute__((unused));				\
 unsigned char *__fsm_ch __attribute__((unused));			\
-TfwStr *__fsm_str __attribute__((unused));
+unsigned int __fsm_chunkflags __attribute__((unused)) = 0;
```

Comment:
The variable is used in interior FSM, so additional interior-specific macro should be added to define variables for interior FSM and do not increase stack size for main FSM.